accepted/2021/statics-in-interfaces/README.md

```diff
@@ -586,66 +639,49 @@ namespace System
 
         static abstract TResult operator *(TSelf left, TOther right);
 
-        static abstract TResult checked operator *(TSelf left, TOther right);
+        static abstract TResult operator checked *(TSelf left, TOther right);
     }
 
-    public interface IDivisionOperators<TSelf, TOther, TResult>
-        where TSelf : IDivisionOperators<TSelf, TOther, TResult>
+    public interface IShiftOperators<TSelf, TOther, TResult>
+        where TSelf : IShiftOperators<TSelf, TOther, TResult>
     {
-        static abstract TResult operator /(TSelf left, TOther right);
+        static abstract TResult operator <<(TSelf value, TOther shiftAmount);
 
-        static abstract TResult checked operator /(TSelf left, TOther right);
+        static abstract TResult operator >>(TSelf value, TOther shiftAmount);
+
+        // Logical right shift
+        static abstract TResult operator >>>(TSelf value, TOther shiftAmount);
     }
 
-    public interface IModulusOperators<TSelf, TOther, TResult>
-        where TSelf : IModulusOperators<TSelf, TOther, TResult>
+    public interface ISubtractionOperators<TSelf, TOther, TResult>
+        where TSelf : ISubtractionOperators<TSelf, TOther, TResult>
     {
-        // The ECMA name is op_Modulus and so the behavior isn't actually "remainder" with regards to negative values
-
-        // Likewise the name IModulusOperators doesn't fit with the other names, so a better one is needed
-
-        static abstract TResult operator %(TSelf left, TOther right);
+        static abstract TResult operator -(TSelf left, TOther right);
 
-        static abstract TResult checked operator %(TSelf left, TOther right);
+        static abstract TResult operator checked -(TSelf left, TOther right);
     }
 
     public interface IUnaryNegationOperators<TSelf, TResult>
```

Comment:
I feel like this should be named `IUnaryMinusOperators` to match `IUnaryPlusOperators`. That's what the operator is called:  https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators#unary-plus-and-minus-operators

Reply:
These match the IL names, not the C# names. The IL name is `op_UnaryNegation`

Reply:
Ah, ok. Although I still think most languages would call it unary minus. Most people don't know IL and don't write it, so if it's called this name exists only in IL, I'm not sure that's a good source to go by.